Added time and event notification modules

fortizpenaloza opened the pull request.

fortizpenaloza committed:
First version

fortizpenaloza committed:
Improved tests [ci skip]

gcotelli commented on source/Kepler-Notifications-Tests/EventNotificationSystemUserStoryTest.class.st:
```diff
@@ -13,6 +13,24 @@ EventNotificationSystemUserStoryTest >> eventNotificationSystem [
 	^ rootSystem >> #EventNotificationSystemInterface
 ]
 
+{ #category : #'as yet unclassified' }
```
There's some methods missing categories here

fortizpenaloza committed:
Added tests for event notification and time modules.
Added support for modules to SystemBasedUserStoryTests.
Added a few comments and improved others.

fortizpenaloza committed:
Merge branch 'release-candidate' into 33-Add-time-and-event-notification-modules
